app/controllers/facility_reservations_controller.rb

```diff
@@ -155,6 +155,7 @@ def update_admin
     set_windows
 
     @reservation.assign_times_from_params(params[:admin_reservation])
+    @reservation.assign_category_attribute(session_user, params[:reservation])
```

Comment:
Is the extra check here on `session_user` necessary? I'm 95% sure you're already validated as being at operator at this point.